main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	// if using go modules
+	"fmt"
+	"net/http"
+
+	"github.com/caarlos0/env/v6"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type ProxyConfig struct {
+	Port int `env:"PORT" envDefault:"3000"`
+
+	// Proxy target
+	RemoteAddress string `env:"REMOTE_ADDRESS"`
+
+	// InputValidation enables the
+	InputValidation bool `env:"INPUT_VALIDATION"`
+	// Allowed Public identities passed in as comma separated b64 encoded
+	// pubkeys
+	// TODO: Will be refactored into a config file
+	AllowedIDs string `env:"ALLOWED_IDS"`
+
+	OutputSigning bool `env:"OUTPUT_SIGNING"`
+	// Private Key is required when output signing is set to true
+	PrivateKeyPath string `env:"PRIVATE_KEY_PATH"`
+
+	// Allow only logs to be proxied
+	LogsOnly bool `env:"LOGS_ONLY" envDefault:"true"`
```

Comment:
In what scenario would we want this used for non-logs?

Reply:
The idea would be for example to allow access to Kibana using the same authentication scheme as used in logs.